packages/debug/src/browser/style/index.css

```diff
@@ -375,6 +375,7 @@
 /** Breakpoint Widget */
 .theia-debug-breakpoint-widget {
     display: flex;
+    height: calc(100% - 4px);
```

Comment:
what for is 4px? we should avoid using hardcoded numbers, is it padding then it should be derived from `var(--theia-ui-padding)`